docs/src/cookbooks/naming.md

```diff
@@ -60,3 +60,36 @@ class ExampleNoPrefix extends MultiIOModule {
 
 println(ChiselStage.emitVerilog(new ExampleNoPrefix))
 ```
+
+### I am still not getting the name I want. For example, inlining an instance changes my name!
+
+In cases where a FIRRTL transform renames a signal/instance, you can use the `forcename` API:
+
+```scala mdoc
+import chisel3.util.experimental.{forceName, InlineInstance}
+
+class WrapperExample extends MultiIOModule {
+  val IN = IO(Input(UInt(3.W)))
+  val OUT = IO(Output(UInt(3.W)))
+  val inst = Module(new Wrapper)
+  inst.IN := IN
+  OUT := inst.OUT
+}
+class Wrapper extends MultiIOModule with InlineInstance {
+  val IN = IO(Input(UInt(3.W)))
```

Comment:
Quibble: All caps names are against (I think) standard style.

Reply:
I agree with Chick here, would prefer lowerCamelCase (just all lower case in this case)